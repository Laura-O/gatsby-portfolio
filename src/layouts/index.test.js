import React from 'react';
import { shallow } from 'enzyme';

import TemplateWrapper from './index';

describe('rendering', () => {
  const props = {
    children: () => 'children',
  };

  it('it renders in production', () => {
    process.env.NODE_ENV = 'production';
    const wrapper = shallow(<TemplateWrapper {...props} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders outside production', () => {
    process.env.NODE_ENV = 'test';
    const wrapper = shallow(<TemplateWrapper {...props} />);
    expect(wrapper).toMatchSnapshot();
  });

  afterEach(() => {
    delete process.env.NODE_ENV;
  });
});
