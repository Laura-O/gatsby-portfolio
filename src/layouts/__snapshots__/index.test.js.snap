// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<div
  className="outer-container"
>
  <Head />
  <Header />
  <div
    className="page"
  >
    children
  </div>
</div>
`;
